README.md

```diff
@@ -1,26 +1,39 @@
-# 🔌 create-eth Extensions
+# Ponder Extension for Scaffold-ETH2
 
-This repository holds all the BG curated extensions for [create-eth](https://github.com/scaffold-eth/create-eth), so you can extend the functionality of your Scaffold-ETH project.
+This is an [Ponder](https://ponder.sh/) extension [Scaffold-ETH 2](https://scaffoldeth.io/).
 
-## Usage
+## Install
 
-You can install any of the extensions in this repository by running the following command:
+Install using ```npx create-eth```
 
-```bash
-npx create-eth@latest -e <extension-name>
-```
 
-## Available extensions
+## 🚀 Setup Ponder Extension
 
-- [subgraph](https://github.com/scaffold-eth/create-eth-extensions/tree/subgraph): Easily extract data from your contracts and expose it through a GraphQL API.
+### Config
 
-## Create your own extension
+Ponder config (```packages/ponder/ponder.config.ts```) is set automatically from the deployed contracts and using the first blockchain network setup at ```packages/nextjs/scaffold.config.ts```.
 
-You can extend Scaffold-ETH by creating your own extension. To do so, you need to create a new repository with the following structure:
+### Design your schema
 
-`ToDo`
+You can define your Ponder data schema on the file at ```packages/ponder/ponder.schema.ts``` following the Ponder documentation (https://ponder.sh/docs/schema).
 
-```bash
-npx create-eth@latest -e your-github-username/your-extension-repository:branch-name # branch-name is optional
-```
+### Indexing data
+
+You can index events by adding files to ```packages/ponder/src/``` (https://ponder.sh/docs/indexing/create-update-records)
+
+### Start the development server
+
+Run ```yarn ponder:dev``` to start the Ponder development server, for indexing and serving the GraphQL API endpoint at http://localhost:42069
+
+### Query the GraphQL API
+
+With the dev server running, open http://localhost:42069 in your browser to use the GraphiQL interface. GraphiQL is a useful tool for exploring your schema and testing queries during development. (https://ponder.sh/docs/query/graphql)
+
+You can query data on a page using ```urql```. Check the code at ```packages/nextjs/app/greetings/page.ts``` to get the greetings updates data and show it.
+
+### Deploy
+
+To deploy the Ponder indexer please refer to the Ponder Deploy documentation https://ponder.sh/docs/production/deploy
```

Comment:
Have you tried to deploy your app? Is it working?

Reply:
I deployed another similar app, but not on a newly created project with the extension, I will deploy it and check that everything is ok!

Reply:
@rin-st Added two lines to README about deployment. 

The app is deployed to https://test-ponder-9zm2plrtx-damianmartigmailcoms-projects.vercel.app/greetings using Ponder deployed to https://test-ponder-extension-production.up.railway.app/

Reply:
<img width="885" alt="image" src="https://github.com/user-attachments/assets/f5f73ab4-998f-4340-87ea-8941842a8867">
Don't have an access to vercel app. But it's not a problem, since I see that Ponder app works!

Lgtm! I will also try to create an app with ponder next days! Thank you!